### ISSUE TITLE
Synthesize Forecaster Research

### ISSUE THREAD

**Issue by colinmurphy01** [weather-gov/weather.gov#215]
Acceptance Criteria
- [x] Understand forecaster experience during a hazard currently
- [x] Understand their goals for each step in relation to user needs
- [ ] BONUS: Understand differences between forecaster experiences across offices
- [x] BONUS: Understand time involved in using existing tools  
- [x] BONUS: Understand forecaster needs for a CMS authoring tool / creating content
- [x] Synthesis is an unbiased representation of trends / themes of what we heard

Considerations:
- [x] We can share findings across the team for discussion (Cowork next friday?)
- [x] We can share findings with Champions in weekly ship 

## Findings
Updated stakeholder journey with added steps and goals for each stage: 
![Screenshot 2023-10-04 at 9 51 11 AM](https://github.com/weather-gov/weather.gov/assets/127144183/301f9cc3-3cbf-4487-9103-ff6024fc7cdf)

Top challenges: 
- Different timelines/thresholds for different partners
- Different tone and detail for partners vs public 
- Difficult to provide all users with relevant information, especially if there are multiple hazards in the coverage area
- Tools to provide messaging around hazards are limiting, time consuming, and can require specific tech skills
- Users can't easily find the information we provide 
- Some offices don't have the same tools available 
- Coordinating with neighboring WFOs that have different criteria for hazard issuance 

Most time-consuming actions: 
- Creating graphics 
- Creating new headlines 
- Updating messaging in CMS and multiple other channels 
- Creating alt text/making images and posts accessible  

[Detailed synthesis available here](https://docs.google.com/document/d/1ZXLohW4RxCOwGliwNlJVrU7vgQEwjAP_BeUKDj_hRes/edit?usp=sharing)

**Comment by kmranjo** [comment]
Initial synthesis [captured in this doc](https://docs.google.com/document/d/1ZXLohW4RxCOwGliwNlJVrU7vgQEwjAP_BeUKDj_hRes/edit). Going to work next on either a writeup or mural (?) to address the AC listed above.

**Comment by kmranjo** [comment]
Updated to reflect the findings as discussed with the team. I think the only thing we can say we haven't quite gotten enough detail on is the bonus item around time spent in different tools. We know which ones take up more time, but not the specific amounts. 
@colinmurphy01  Am I correct that the remaining items for this are to get team alignment on action items/next steps? I've added some initial ideas to the document based on our convo last week, but our team conversation didn't get that far. Is this something we might want to talk about in standup? Or is there a better venue for that?
